src/components/Common/Fields/ComboBoxField/ComboBoxField.tsx

```diff
@@ -0,0 +1,43 @@
+import { useField, type UseFieldProps } from '@/components/Common/Fields/hooks/useField'
+import { ComboBox } from '@/components/Common/UI/ComboBox/ComboBox'
+
+interface ComboBoxItem {
+  id: string
+  name: string
+}
+
+interface ComboBoxFieldProps
+  extends Omit<React.ComponentProps<typeof ComboBox>, 'name' | 'onChange' | 'onBlur' | 'options'>,
+    UseFieldProps<string> {
+  items: ComboBoxItem[]
+}
+
+export const ComboBoxField: React.FC<ComboBoxFieldProps> = ({
+  rules,
+  defaultValue,
+  name,
+  errorMessage,
+  isRequired,
+  onChange,
+  transform,
+  items,
+  ...comboBoxProps
+}: ComboBoxFieldProps) => {
+  const fieldProps = useField({
+    name,
+    rules,
+    defaultValue,
+    errorMessage,
+    isRequired,
+    onChange,
+    transform,
+  })
+
+  return (
+    <ComboBox
+      {...fieldProps}
+      options={items.map(item => ({ label: item.name, value: item.id }))}
+      {...comboBoxProps}
```

Comment:
Could we update this so that ComboBoxField also uses `options` instead of mapping to a separate `items` type? I'm +1 to options as implemented in ComboBox for the consistency with Select and think ComboBoxField would also benefit from using that so it's not a different prop using one over the other

Reply:
Updated - WIll be in next revision